docs/release-notes/2.40.0.md

```diff
@@ -113,3 +113,16 @@ VictoriaMetrics has been upgraded [v1.93.4](https://docs.victoriametrics.com/CHA
 ## Known issue
 
 - [PMM-12517](https://jira.percona.com/browse/PMM-12517) - If you have set custom dashboards as your Home dashboard in version 2.26.0, they may be lost after upgrading to version 2.40.0. To prevent this, we recommend that you back up your custom dashboard before upgrading and recover it after the upgrade.
+
+- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading PMM from version 2.39.0 to 2.40.0 (not el7) using Docker, the `admin` user cannot access the PMM UI.
+
+    **Solution**: To fix the problem and gain back admin access to the PMM interface execute the following:
+
+    ```sh
+    # psql -U grafana
+    grafana=> update "user" set id='1' where login='admin';
+    UPDATE 1
+    grafana=> \q
+
+    # grafana-cli --homepath=/usr/share/grafana --config=/etc/grafana/grafana.ini admin reset-admin-password <PASS>
```

Comment:
Maybe link to another section to not duplicate the command?

Reply:
fixed